apps/masterbots.ai/components/routes/browse/browse-chat-messages.tsx

```
@@ -53,11 +62,27 @@ export function BrowseChatMessages({
 
+  // Fetch parent thread info
+  const fetchParentThreadInfo = async () => {
+    if (parentThreadId) {
+      const parentThread = await getMessages({ threadId: parentThreadId })
+      const parentThreadTitle = parentThread[0]?.content
+      setParentThreadTitle(parentThreadTitle)
+    }
+  }
+
   // Effect to fetch messages when the thread ID changes
   React.useEffect(() => {
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: fetchMessages

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)